docs/platforms/react-native/migration/v5-to-v6.mdx

```diff
@@ -0,0 +1,253 @@
+---
+title: Migrate from 5.x to 6.x
+sidebar_order: 100
+description: "Learn how to migrate from version 5 to version 6 of the Sentry React Native SDK"
+---
+
+The main goal of version 6 of the Sentry React Native SDK, is to bring compatibility with Sentry JavaScript version 8 and to improve our performance monitoring APIs and integrations. This version is breaking because of the JavaScript SDK dependency upgrade to version 8 and because we've removed deprecated APIs and restructured npm package contents.
+
+## Important Changes In Dependencies
+
+The Sentry React Native SDK ships with the Sentry JavaScript SDK as a dependency. The Sentry JavaScript SDK has been updated to version 8. This version includes a number of breaking changes. Please follow the JavaScript migration guides to complete the upgrade.
+
+The initial `@sentry/react-native@6.0.0` ships with `@sentry/core@8.31.0`. Always use the exact JavaScript SDK version if adding it manually to your project. Any other version might not be compatible with the React Native SDK.
+
+### Performance-Monitoring API Changes
+
+The main focus of the Sentry JavaScript SDK version 8, is to improve our performance monitoring APIs. That's why the Tracing API changes are described in a standalone guide. Please follow the [Tracing API changes](/platforms/javascript/migration/v7-to-v8/v8-new-performance-api/) guide to complete the upgrade.
+
+### General API Changes
+
+Other general JavaScript SDK version 8 changes are described in the [JavaScript SDK 7.x to 8.x migration guide](/platforms/javascript/migration/v7-to-v8/).
+
+## Important React Native SDK `6.x` Changes
+
+This section describes API changes in the Sentry React Native SDK, version 6.
+
+### React Native Tracing Options Moved to `Sentry.init` Options
+
+To change the following options, you no longer need to create an instance of React Native Tracing. Instead, you can pass them directly to `Sentry.init` options.
+
+```javascript diff
+import * as Sentry from '@sentry/react-native';
+
+Sentry.init({
+  tracesSampleRate: 1.0,
++  enableAppStartTracking: true,
++  enableNativeFramesTracking: true,
++  enableStallTracking: true,
++  enableUserInteractionTracing: true,
+-  integrations: [new Sentry.ReactNativeTracing({
+-    enableAppStartTracking: true,
+-    enableNativeFramesTracking: true,
+-    enableStallTracking: true,
+-    enableUserInteractionTracing: true,
+-  })],
+});
+```
+
+### Navigation Instrumentations Are Now Standalone Integrations
+
+Navigation Instrumentations are now standalone integrations. You'll need to add it to the `integrations` array in the `Sentry.init` options.
+
+```javascript {tabTitle:Expo Router} diff
+import Sentry from '@sentry/react-native';
+import { useNavigationContainerRef } from 'expo-router';
+
+- const reactNavigationIntegration = new Sentry.ReactNavigationInstrumentation();
++ const reactNavigationIntegration = new Sentry.reactNavigationIntegration();
```

Comment:
We are not using this format for V6, see:
https://github.com/getsentry/sentry-react-native/pull/3851/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR117


```suggestion
+ const reactNavigationIntegration =  Sentry.reactNavigationIntegration();
```